code/datums/components/crafting/crafting.dm

```diff
@@ -42,6 +42,9 @@
 					CAT_SANDWICH,
 					CAT_SOUP,
 					CAT_SPAGHETTI,
+		// SKYRAT ADDITION BEGIN - Teshari food category
+					CAT_TESHARI,
+		// SKYRAT ADDITION END
```

Comment:
I'm not saying this is bad, but we do need to find a better way of modularly doing this!